services/userService.js

```diff
@@ -0,0 +1,61 @@
+const User = require("../models/user");
+
+async function createUser(data) {
+  try {
+    const user = await User.create(data);
+    return user;
+  } catch (error) {
+    throw error;
+  }
+}
+
+async function getUserById(id) {
```

Comment:
The function declaration is incomplete. It should have a function body and a return statement. Additionally, it should handle potential errors. Consider implementing a try-catch block to handle any errors that may occur during the execution of the function.